src/EA.Iws.Web/Controllers/ApplicantController.cs

```diff
@@ -1,15 +1,13 @@
 ﻿namespace EA.Iws.Web.Controllers
 {
-    using System;
+    using Infrastructure.Authorization;
+    using System.Linq;
     using System.Threading.Tasks;
     using System.Web.Mvc;
-    using Core.NotificationAssessment;
     using Prsd.Core.Mediator;
     using Requests.Notification;
-    using Requests.NotificationAssessment;
-    using ViewModels.Applicant;
```

Comment:
using Infrastructure.Authorization; shouldn't be at the top. Not sure why StyleCop hasn't picked that up!